docs/source/introduction.rst

```diff
@@ -0,0 +1,39 @@
+.. index:: setup
+
+************
+Introduction
+************
+
+
+.. index:: motivation
+
+Motivation
+==========
+
+
+.. index:: additional information
+
+Additional information
+======================
+
+Euro Coin
+---------
+In order to design our coin box we will take into account euro coin characterisitcs. 
+
+- The maximum diameter is 25.75 mm for 2 euro coin
+- The minimum diameter is 16.25 mm for 1 cent coin
+- The maximum edge thickness is 2.38 mm for 50 cent
+- The minimum edge thickness is 1.67 mm for 1, 2 and 5 cents
+
+About composition:
+
+- €2: outer ring - cupronickel; inner section (three layers) - nickel-brass, nickel, nickel-brass (slightly magnetic)
+- €1: outer ring - nickel-nickel, inner section (three layers) - cupronickel, nickel, cupronickel (slightly magnetic)
+- 50 cent, 20 cent and 10 cent: copper, aluminium, zinc, tin alloy = "Nordic gold" (non-magnetic)
+- 5, 2 and 1 cent: copper-coated steel (highly magnetic)
```

Comment:
I'd rather use sublists or say "The outer ring is composed... . The inner section has three layers, the first..., the second...".

Reply:
new pull request for documentation